r4e-img-upgrade-iso.sh

```diff
@@ -0,0 +1,58 @@
+# script for creating an update ISO
+# the image includes the updated agent RPM that communicates with server that monitors edge devices
+# at the end an upgraded ostree image will be available
+# input:
+#  - original-image-name that was chosen for the first step (Build ISO for edge devices)
+#  - parrent-ostree-commit can be found in the original image webserver under in compose.json file under the key 'ostree-commit'
+#  - image-version-number the new version of the image
+# script assumes that file `blueprint.toml` is present and updated in the working directory
+set -e
+Usage() {
+    echo "Usage:"
+    echo `basename $0` "<original-image-name> <parrent-ostree-commit> <image-version-number>"
+}
+
+# check usage
+if [ $# != 3 ] ; then
+    echo -e "bad usage\n"
+    Usage
+    exit 1
+fi
+
+# read input
+export IMAGE_NAME=$1
+export OSTREE_COMMIT=$2
+export IMAGE_VERSION=$3
+export NEW_IMAGE_NAME=${IMAGE_NAME}-${IMAGE_VERSION}
+
+# prepare local variables
+IMAGE_FOLDER=/var/www/html/${NEW_IMAGE_NAME}
+BLUEPRINT_FILE=blueprint.toml
+BLUEPRINT_NAME=${IMAGE_NAME}
+
+
+# create a new blueprint
+composer-cli blueprints push $BLUEPRINT_FILE
+
+# validate blueprint
+echo "validating blueprint $BLUEPRINT_NAME"
+composer-cli blueprints depsolve $BLUEPRINT_NAME
```

Comment:
if dependencies are missing will it this command fail and terminate the entire script ?

Reply:
No thanks I will change it